docs/getting_started/tutorials/conversion/index.md

```diff
@@ -0,0 +1,3 @@
+# BIDS Conversion
+
+Converting data to BIDS organization and naming is typically the largest barrier to using BIDS tools. The following guides have been created to assist in converting raw imaging data into BIDS-compliant datasets. 
```

Comment:
```suggestion
Converting data to BIDS organization and naming is typically the largest barrier to using BIDS tools. The following guides have been created to assist in converting your into BIDS-compliant datasets. 
```
- made some edits above, like 'raw imaging' -> 'your' 
- [ ] see if there's a way to simplify this - what you really mean is "We know that conversion has its challenges, including file re-org and re-naming.  Here are some guides to help. "